resources/build/zip.inc.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+#
+# This script contains zip and unzip (TODO) utilities to function for all environments.
+# unzip is available in all environments
+# zip is not available by default on Windows, but can be manually installed.
+# If zip is not available, then fall back to 7z for zipping.
+# 7z requires env SEVENZ_HOME.
+#
+# TODO: refactor with /resources/build/win/zip.inc.sh
+
+# zip/7z to create an archive with the following parameters (in order)
+# [zip filename]
+# [list of flags to pass to zip command. Flags start with a single-dash
+#   -x@filename for a file containing list of files to exclude from the archive
+#   -* all other flags]
+# [list of files to include in zip]
+function zip_files() {
+
+  # Parse parameters
+
+  # $1 for zip filename
+  local ZIP_FILE="$1"
+  shift
+
+  # Parse rest of parameters
+  EXCLUDE=
+  FLAGS=()
+  INCLUDE=()
+  while [[ $# -gt 0 ]] ; do
+    case "$1" in
+      -x@*)
+        # filename of files to exclude
+        EXCLUDE="$1"
+        shift
+        ;;
```

Comment:
I'm still not sure why we have special-case this? `zip` apparently allows `-x` before files and so too does `7z`?

Reply:
Yeah, I can remove now that we can treat -x@ as a regular flag.

---

Looking ahead (separate PRs) to flags for 7z-isms 
Windows builds are setting compression method 
> -mx1 (fastest) .. -mx9 (ultra)

https://github.com/keymanapp/keyman/blob/ed33566d3c5010af2baa48e51d90e480d56e6eae/resources/build/win/zip.inc.sh#L5-L13

Reply:
The GO_FAST is useful for speeding up test builds -- reduces time for the installer build by several minutes. We can tweak the script when the time comes to support that -- the pattern you have here allows for that.